src/Features/Core/Portable/Shared/TestHooks/Legacy/AggregateAsynchronousOperationListener.cs

```diff
@@ -1,38 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Roslyn.Utilities;
-using System.Runtime.CompilerServices;
-
-namespace Microsoft.CodeAnalysis.Shared.TestHooks
-{
-    [Obsolete("use IAsynchronousOperationListenerProvider")]
-    internal class AggregateAsynchronousOperationListener : IAsynchronousOperationListener
```

Comment:
Wasn't this a shim you left for TypeScript to continue working? Did they move off of it now?

Reply:
yes, it was shims. forgot to tag them. but due to [Obsolete], they should have build break if they didn't move. also actual waiter was deleted way before. 

@amcasey

Reply:
FYI @uniqueiniquity

Reply:
Just moved, should be good to go.